src/components/interface/LogoutModal.vue

```diff
@@ -12,18 +12,18 @@
         </div>
       </div>
 
-      <div class="flex items-center pb-4 mx-4">
+      <div class="mr-4 text-center">
         <Button
           color="green"
-          class="w-full p-4 mt-4"
+          class="w-full md:w-3/4 p-4 m-4 m-auto"
```

Comment:
Alignment/spacing is a bit off before `md` breakpoint

<img width="524" alt="Screen Shot 2021-09-22 at 13 30 00" src="https://user-images.githubusercontent.com/22848332/134417167-dd5a0acc-5fa1-4f91-9684-75b3ca5815bf.png">

Reply:
Adjusted margin for smaller screens and to be consistent with confirmation modal